fhir_transformer/src/main.py

```diff
@@ -59,13 +65,46 @@ def manage_kafka_error(msg):
     logger.error(msg.error())
 
 
-if __name__ == "__main__":
-    logger.info("Running Consumer")
+def transorm_row(resource_id, df):
```

Comment:
typo

Reply:
+ it is not a dataframe but a dict so I would suggest to change the naming

Reply:
I called it dataframe because it was called that way in the kafka event. renamed it to `row`